Dockerfile

```diff
@@ -28,7 +28,7 @@ ENV PYTHON_VER=python3.11
 RUN apt update && \
 	apt-get install -y software-properties-common && \
 	add-apt-repository ppa:deadsnakes/ppa && \
-	apt install -y $PYTHON_VER python3-pip && \
+	apt install -y $PYTHON_VER python3-pip gcc $PYTHON_VER-dev && \
```

Comment:
What is gcc needed for? There's no way to do the build in the build image and copy the result over?

Reply:
removed gcc.